aiohttp/helpers.py

```diff
@@ -737,3 +755,10 @@ def content_length(self, *, _CONTENT_LENGTH=hdrs.CONTENT_LENGTH):
 
         if content_length:
             return int(content_length)
+
+    @property
+    def content_disposition(self, *, header=hdrs.CONTENT_DISPOSITION):
```

Comment:
Why the property is in `HeadersMixin`?
Should `web.Request` has the property?
I thought it is only for `ClientResponse`

Reply:
Indeed it's response only header.